### PR TITLE
Cleans up dirty baselines

### PR DESCRIPTION
**Pull request by petrblaho**
## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: system_baseline/views/v1.py [hunk 1]
```diff
@@ -313,26 +313,31 @@ def check_dirty_baselines(baselines):
...
+                    except ValueError as error:
+                        message = str(error)
+                        current_app.logger.audit(message, request=request, success=False)
+                    except Exception:
+                        message = "Unknown error when deleting system with baseline"
+                        current_app.logger.audit(message, request=request, success=False)
+
+            baseline.dirty_systems = False
+            db.session.add(baseline)
+            db.session.commit()
```
Comment (vkrizan) [inline system_baseline/views/v1.py:340]: Do you want to commit it for each baseline individually?

**Review by vkrizan — APPROVED** [review]

File: system_baseline/views/v1.py (hunk 1 above)
Comment (petrblaho) [reply system_baseline/views/v1.py:345]: Yes... but https://github.com/RedHatInsights/system-baseline-backend/pull/282/files#diff-285e0634d3b94393071de1f685907e13d80487fa2e250ae076868664473fbad3R333 has commit inside anyway...

Comment (petrblaho) [reply system_baseline/views/v1.py]: I mean, you are right... I changed that.... but the other is still there now...

**Comment by sthirugn** [comment]
please add jira ticket for reference in the future. 

EVNT-696

**Pull request merged**